esrally/mechanic/mechanic.py

```diff
@@ -290,6 +296,10 @@ def receiveMsg_StartEngine(self, msg, sender):
         cls = metrics.metrics_store_class(self.cfg)
         self.metrics_store = cls(self.cfg)
         self.metrics_store.open(ctx=msg.open_metrics_context)
+        name = self.cfg.opts("race", "pipeline")
+        team_path = self.cfg.opts("mechanic", "team.path", mandatory=False)
+        if not (name == "benchmark-only" or team_path):
+            self.team_revision = git.head_revision(team.team_path(self.cfg))
```

Comment:
Similar to my comment below about track-parameters, could we have a cfg argument for `team_revision` -- which could even be specified via cli arguments -- and initialize it, if unset, e.g. in [team_path](https://github.com/elastic/rally/blob/9b9c7e03b79a11414740fa8f543b7a58b162149b/esrally/mechanic/team.py#L128-L147)? The information could then be retrieved from the cfg object here.